docs/content/drafts/UnderlineNav2.mdx

```diff
@@ -104,20 +104,61 @@ render(<Navigation />)
 ### With React Router
 
 ```jsx
-import {Link} from 'react-router-dom'
+import {Link, useMatch, useResolvedPath} from 'react-router-dom'
 import {UnderlineNav} from '@primer/react/drafts'
+
+function UnderlineNavItem({to, children, ...rest}) {
+  const resolved = useResolvedPath(to)
+  const isCurrent = useMatch({path: resolved.pathname, end: true})
+  return (
+    <UnderlineNav.Item as={Link} to={to} aria-current={isCurrent ? 'page' : undefined} {...rest}>
+      {children}
+    </UnderlineNav.Item>
+  )
+}
+
+const Navigation = () => {
+  return (
+    <UnderlineNav aria-label="Repository">
+      <UnderlineNavItem to="/code" counter={4}>
+        Code
+      </UnderlineNavItem>
+      <UnderlineNavItem to="/issues" counter={44}>
+        Issues
+      </UnderlineNavItem>
+      <UnderlineNavItem to="/pulls">Pull Requests</UnderlineNavItem>
+    </UnderlineNav>
+  )
+}
+```
+
+### With Next.js
+
+```jsx
```

Comment:
Shouldn't we add "```jsx live drafts" here as well?

Reply:
Hmm, because they won't actually work on the docs because there is no react router here, so I only used jsx. Do I miss anything here?

Reply:
You are right! Forget my comment then 😅

Reply:
All good, thanks for the review! 🎉